scripts/check.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# Basic instancetype.kubevirt.io version check
+INSTANCETYPE_API_VERSION="v1beta1"
+if grep -ri instancetype.kubevirt.io/v1 common-instancetypes | grep -v ${INSTANCETYPE_API_VERSION}; then
```

Comment:
Might be enough to grep just for `instancetype.kubevirt.io/`?

Reply:
That picks up various labels:

```
$ git grep instancetype.kubevirt.io/
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:apiVersion: instancetype.kubevirt.io/v1beta1
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/class: Compute Exclusive
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/description: |-
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/version: "1"
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/cpu: "8"
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/dedicatedCPUPlacement: "true"
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/hugepages: "true"
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/isolateEmulatorThread: "true"
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/memory: 16Gi
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/numa: "true"
data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml:    instancetype.kubevirt.io/vendor: kubevirt.io
```

Reply:
`apiVersion: instancetype.kubevirt.io/`?

Reply:
I've gone with `grep -riP "instancetype.kubevirt.io/v\d+" common-instancetypes`.